README.md

```diff
@@ -1,2 +1,13 @@
-# P1.9_Collaborative_Project
-This repo contains the collaborative project of the MHPC course P1.9
+This package contains simplified MD code with multi-threading
```

Comment:
Don't get rid of the repo title. Try to create a new heading for compilation instructions